app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.github.weg_li_android">
-
+    <uses-feature android:name="android.hardware.camera"
+        android:required="false" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Müssen wir wirklich im externen storage speichern? Können wir das nicht app-intern tun? Note: Bisher hab ich noch nie Fotos aufgenommen innerhalb einer App, hab da also keine Erfahrung :D

Reply:
I was debating about this as well. I came to the conclusion that in some cases it might be useful to have to photos taken with the app also on external storage. For example you can't finish the transmission (due to network connection) and you want to do it with weg-li website mass-upload later.